Decred Wallet/table_view_cell/BaseTableViewCell.swift

```diff
@@ -42,7 +42,7 @@ open class BaseTableViewCell : UITableViewCell {
             self.textLabel?.text = menuText
             if (menuText == LocalizedStrings.overview) {
                 self.imageView?.image = UIImage(named: "overview")
-            } else if (menuText == LocalizedStrings.accounts) {
+            } else if (menuText == LocalizedStrings.wallets) {
```

Comment:
Is this file still necessary?

Reply:
not anymore. removed in 37e2453. all related functions used in other table view cells have been moved to `Decred Wallet/Extensions/UITableView.swift`